### PR TITLE
fix(Agent UI): Displays incorrect version of Agent OS type

### PR DESCRIPTION
**Pull request by manikanta-reddy-t**

### CODE REVIEW

File: ui/src/app/components/webcomponents/lab-environments-info.component.ts [hunk 1]
```diff
@@ -65,7 +65,7 @@ export class LabEnvironmentsInfoComponent implements OnInit {
           if(res.content.length){
             agent = res.content[0];
             if(env.browser!=null){
-              env.platform = Agent.getPlatformFromOsType(agent.osType);
+              env.platform = agent.getPlatformFromOsType(agent.osType);
```
Comment (guganabu) [inline ui/src/app/components/webcomponents/lab-environments-info.component.ts:68]: other modules have agent in class properties but here it's captured in variable, plz make it consistency

File: ui/src/app/components/webcomponents/test-plan-platform-os-version-form.component.ts [hunk 2]
```diff
@@ -92,8 +92,13 @@ export class TestPlanPlatformOsVersionFormComponent implements OnInit {
...
         }
         if (this.platform == null) {
-          this.platform = this.platforms[0];
-          this.environmentFormGroup?.controls['platform']?.setValue(this.platform?.id);
+          if(!this.isHybrid){
+            this.platform = this.platforms[0];
+            this.environmentFormGroup?.controls['platform']?.setValue(this.platform?.id);
+          } else if (this.version.workspace.isWeb) {
+            this.platform = this.platforms.find(platform=>platform.name == this.agent.getPlatformFromOsType(this.agent.osType))
+            this.environmentFormGroup?.controls['platform']?.setValue(this.platform.name);
```
Comment (guganabu) [inline ui/src/app/components/webcomponents/test-plan-platform-os-version-form.component.ts:100]: platform formControl value either id/value?

File: ui/src/app/components/webcomponents/test-plan-platform-os-version-form.component.ts [hunk 3]
```diff
@@ -152,7 +157,7 @@ export class TestPlanPlatformOsVersionFormComponent implements OnInit {
       this.browsers = res;
       if (!this.isPrivateGrid) {
         if (this.environmentFormGroup?.controls['browser']?.value)
-          this.platformBrowser = this.browsers.find(browser => browser.id == this.environmentFormGroup?.controls['browser']?.value);
+          this.platformBrowser = this.browsers.find(browser => browser.name.toLowerCase().includes(this.environmentFormGroup?.controls['browser']?.value.toLowerCase()) );
```
Comment (guganabu) [inline ui/src/app/components/webcomponents/test-plan-platform-os-version-form.component.ts:160]: what is the need of includes here?

File: ui/src/app/components/webcomponents/test-plan-platform-os-version-form.component.ts (hunk 2 above)
Comment (manikanta-reddy-t) [reply ui/src/app/components/webcomponents/test-plan-platform-os-version-form.component.ts:100]: here both id and name returns same value.

File: ui/src/app/components/webcomponents/lab-environments-info.component.ts (hunk 1 above)
Comment (manikanta-reddy-t) [reply ui/src/app/components/webcomponents/lab-environments-info.component.ts:68]: Done, added agent property to the class

File: ui/src/app/components/webcomponents/test-plan-platform-os-version-form.component.ts (hunk 3 above)
Comment (manikanta-reddy-t) [reply ui/src/app/components/webcomponents/test-plan-platform-os-version-form.component.ts:160]: i think not need, we can check using ' == ' only

**Review by guganabu — APPROVED** [review]

**Pull request merged**